lanretam/settings_shared.py

```diff
@@ -44,7 +44,7 @@
 ]
 
 THUMBNAIL_SUBDIR = "thumbs"
-LOGIN_REDIRECT_URL = "/"
+LOGIN_REDIRECT_URL = "/cms/"
```

Comment:
You're changing LOGIN_REDIRECT_URL here, but below you're looking up LOGOUT_REDIRECT_URL. Is there a typo here?

Reply:
I made the fix in the PR following this